docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,208 @@
+# Quick Start with Terraform
+
+This tutorial aims at presenting how to set up an environment with Liqo installed via Terraform and explore the Liqo capabilities through its Provider and Resources.
+
+## Provision the infrastructure
+
+Before starting the tutorials below, you should ensure the following software is installed on your system:
+
+- [**Docker**](https://docker.io), the container runtime.
+- [**Kubectl**](https://kubernetes.io/docs/tasks/tools/install-kubectl/), the command-line tool for Kubernetes.
+- [**Terraform**](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli), the command-line tool for Terraform.
+
+Then, let's open a terminal on your machine and launch the following script, which creates the complete infrastructure with all playground already set up.
+
+{{ env.config.html_context.generate_clone_example_tf('quick-start-terraform') }}
+
+## Analyze the infrastructure and code
+
+With the previous command you created two KinD clusters, peered with each other. Furthermore, you offloaded a namespace in Liqo virtual cluster.\
```

Comment:
```suggestion
The previous command created two KinD clusters, peered with each other. Furthermore, you offloaded a namespace in Liqo virtual cluster.\
```
Be more specific. How are resources offloaded? A offloads to B? Viceversa? Bi-directional?